Ray.jl/test/runtests.jl

```diff
@@ -0,0 +1,9 @@
+include("set_up_tests.jl")
```

Comment:
Better name? Is there a precedent for this?
```suggestion
include("import_deps.jl")
```

Reply:
yeah, this is a beacon package templates thing to make running tests locally less painful.  but I'm not sure it's needed here (no juggling AWS permissiosn or anything of the sort)

Reply:
I removed this in #19